src/install/install.php

```diff
@@ -406,17 +419,18 @@ private function install(): bool
     private function getConfigOutput(): string
     {
         // Version data
-        $version = new \FOSSBilling\Requirements();
         $reg = '^(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)(?:-(?P<prerelease>(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+(?P<buildmetadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$^';
         $updateBranch = (preg_match($reg, \FOSSBilling\Version::VERSION, $matches) !== 0) ? "release" : "preview";
 
         // Load default sample config
         $data = require PATH_CONFIG_SAMPLE;
 
         // Handle dynamic configs
-        $data['security']['force_https'] = FOSSBilling\Tools::isHTTPS() ? true : false;
+        $data['security']['force_https'] = FOSSBilling\Tools::isHTTPS();
+        $data['debug_and_monitoring']['report_errors'] = (bool)$this->session->get('error_reporting');
         $data['update_branch'] = $updateBranch;
-        $data['salt'] = md5(random_bytes(13));
+        $data['info']['salt'] = bin2hex(random_bytes(16));
```

Comment:
Just wanted to make note of the change on how the salt is generated from `md5(random_bytes(13));` to `bin2hex(random_bytes(16));`.

It's unrelated to this PR, however I was already making changes on this line and I've wanted to change this for awhile now to be a better method. The new way still produces the same length of a salt as before (not that it matters), but actually has slightly more entropy in it.